packages/core/lib/settings/config.dart

```diff
@@ -47,13 +47,19 @@ flutter_gen:
     enabled: true
     package_parameter_enabled: false
     style: dot-delimiter
+    outputs:
+      class_name: Assets
     
   fonts:
     enabled: true
+    outputs:
+      class_name: FontFamily
 
   colors:
     enabled: true
     inputs: []
+    outputs:
+      class_name: ColorName
```

Comment:
Same above comment becoz you checked in this line https://github.com/FlutterGen/flutter_gen/pull/280/files#diff-4f1088f5086a316577b245788f11453e91629f92b823d1fa58936750f6f1f518R26